formatTest/unit_tests/expected_output/wrappingTest.re

```diff
@@ -252,62 +252,56 @@ let defOptionalAliasAnnot =
   10;
 
 /* Invoking them */
-named
+named(
   /* a::a */
-  (
-    ~a,
-    /* b::b; */
-    ~b
-  );
+  ~a,
+  /* b::b; */
+  ~b
+);
 
-named
+named(
   /* a::a */
-  (
-    ~a,
-    /* b::b; */
-    ~b
-  );
+  ~a,
+  /* b::b; */
+  ~b
+);
 
-optional
+optional(
   /* a::a */
-  (
-    ~a,
-    /* b::b; */
-    ~b
-  );
+  ~a,
+  /* b::b; */
+  ~b
+);
 
-optional
+optional(
   /* a::a */
-  (
-    ~a,
-    /* b::b; */
-    ~b
-  );
+  ~a,
+  /* b::b; */
+  ~b
+);
 
 let explictlyPassed =
   /* optional */
-  optional
+  optional(
     /* a::? */
     /* None */
-    (
-      ~a=?None,
-      /* b::? */
-      /* None; */
-      ~b=?None
-    );
+    ~a=?None,
+    /* b::? */
+    /* None; */
```

Comment:
This one is a little worse, but it's much less common that the other things that this diff improves. In future formatting fixes, we hope that even this one will be improved.